src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml

```diff
@@ -44,6 +44,7 @@
     <!--[if IE 8]><link href="@Styles.Url("~/bundles/screenie8")" media="screen" rel="stylesheet" /><![endif]-->
     <!--[if gt IE 8]><!-->
     <link href="@Styles.Url("~/bundles/screen")" media="all" rel="stylesheet" />
+    <link href="@Html.CdnLink("css", "main.css")" rel="stylesheet" />

```

Comment:
We don't want to pull the styles from the CDN, work is being done to copy the styles required into the local stylesheet for now.

Reply:
Should I remove?  This story will be blocked until that work is complete.